src/resolver.cpp

```diff
@@ -24,6 +21,7 @@
 #include <openassetio/log/LoggerInterface.hpp>
 #include <openassetio/log/SeverityFilter.hpp>
 #include <openassetio/python/hostApi.hpp>
+#include <openassetio/trait/TraitsData.hpp>
```

Comment:
[Fix] Sorry no where else to put this, be good to add a `hasCapability` check at L211